feature/floor-map/src/main/java/io/github/droidkaigi/confsched2023/floormap/component/FloorMapSideEventItem.kt

```diff
@@ -1,18 +1,149 @@
+import io.github.droidkaigi.confsched2023.model.SideEvent.Mark
+import io.github.droidkaigi.confsched2023.model.SideEvent.Mark.Mark1
+import io.github.droidkaigi.confsched2023.model.SideEvents
 
 @Composable
 fun FloorMapSideEventItem(
     sideEvent: SideEvent,
     onSideEventClick: (SideEvent) -> Unit,
     modifier: Modifier = Modifier,
 ) {
-    Column(modifier) {
-        Text(sideEvent.toString())
+    Column(
+        modifier
+            .clickable { onSideEventClick(sideEvent) }
+            .padding(horizontal = 16.dp)
+    ) {
+        Spacer(modifier = Modifier.height(16.dp))
+        Row(verticalAlignment = Alignment.CenterVertically) {
+            val (iconVector, iconColor) = sideEvent.mark.iconResAndColor()
+            Icon(
+                imageVector = iconVector,
+                contentDescription = FavoriteIcon.asString(),
+                tint = iconColor,
+                modifier = Modifier.size(16.dp),
+            )
+            Spacer(modifier = Modifier.width(8.dp))
+            Text(
+                text = sideEvent.title.currentLangTitle,
+                style = MaterialTheme.typography.labelLarge,
+                maxLines = 1,
+                overflow = TextOverflow.Ellipsis,
+            )
+        }
+        Spacer(modifier = Modifier.height(4.dp))
+        Text(
+            text = sideEvent.description.currentLangTitle,
+            style = MaterialTheme.typography.bodyMedium,
+        )
+        Spacer(modifier = Modifier.height(8.dp))
+        Row(verticalAlignment = Alignment.CenterVertically) {
+            Icon(
+                imageVector = Icons.Filled.Schedule,
+                contentDescription = FavoriteIcon.asString(),
+                modifier = Modifier.size(16.dp)
+            )
+            Spacer(modifier = Modifier.width(8.dp))
+            Text(
+                text = sideEvent.timeText.currentLangTitle,
+                style = MaterialTheme.typography.bodySmall,
+            )
+        }
+        if (sideEvent.link != null) {
+            Spacer(modifier = Modifier.height(8.dp))
+            Row(verticalAlignment = Alignment.CenterVertically) {
+                Icon(
+                    imageVector = Icons.Filled.Link,
+                    contentDescription = FavoriteIcon.asString(),
+                    modifier = Modifier.size(16.dp)
+                )
+                Spacer(modifier = Modifier.width(8.dp))
+                Text(
+                    text = createAnnotatedEventDetailString(),
+                    style = MaterialTheme.typography.bodySmall,
+                    modifier = Modifier.clickable {
+                        // todo open link
+                    }
+                )
+            }
+        }
+        Spacer(modifier = Modifier.height(16.dp))
+        Divider()
+    }
+}
+
+@Composable
+private fun Mark.iconResAndColor(): Pair<ImageVector, Color> {
+    return when (this) {
+        Mark1 -> Icons.Filled.Favorite to LocalContentColor.current
```

Comment:
I tried to define the color of the icon, but couldn't find the right place.
For example, I thought it would be good to define pink in designsystem/theme/Color.kt? (It didn't seem to be visible in the visibility of the module)
In the sense of temporary, if you directly define #DC369A here, it will be resolved.

Reply:
I would like to have it in Mark properties. This is because it is better that the member of adding SideEvent doesn't have to know this when expression

```kotlin
enum MarkColors {
  Pink,
  ...
}
```

Reply:
@takahirom 
Thanks！
But I don't fully understand.

like this?

```
SideEvent {...
    enum class Mark(val color: MarkColor) {
        Mark1(Pink),
    }

    enum class MarkColor {
        Pink;
    }
```

```
// In FloorMapSideEventItem.kt

            Icon(
                imageVector = iconVector,
                contentDescription = FavoriteIcon.asString(),
                tint = sideEvent.mark.color,
                modifier = Modifier.size(16.dp),
            )
```
sideEvent.mark.color is just enum.
We have to associate the enum with the color, should I associate it with the FloorMapSideEventItem?

```
// In FloorMapSideEventItem.kt(In some function)
MarkColor.Pink -> Color(0xFFF8FAF6)
```

I think it would be better to unify the pink color to one value, but is it okay to define the value (0xFFF8FAF6) directly in the "Floor map module"?

Reply:
All I want to do is that DroidKaigi staff can add an side event easily.
I though they should modify here when they do it.
I'll give it a thought.👀

Reply:
I was wondering if you could create an issue for binding Marks and Color and icons. 🙇 
I realized this is what we have to do.

Reply:
I've got it and I agree.
But sorry I just push 0b02a52 and a342450.

And I think it' better bind ColorObject and ColorHex which doesn't define at Color.kt companion object to avoid different modules may have different pink colors.(I just come up with an idea)